perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/TasksManagerBlImpl.java

```diff
@@ -41,6 +41,8 @@
 	@Autowired
 	protected TasksManagerImplApi tasksManagerImpl;
 
+	private boolean suspendedTasksPropagation = false; //are tasks stopped from propagating to engine
```

Comment:
TaskManager should be singleton in a Spring config, but you probably want this to be static variable.